addons/sys_core/XEH_postInit.sqf

```diff
@@ -17,6 +17,8 @@ if (!hasInterface) exitWith {};
 ["localStopSpeaking", FUNC(localStopSpeaking)] call EFUNC(sys_rpc,addProcedure);
 ["pong", FUNC(pong)] call EFUNC(sys_rpc,addProcedure);
 ["gen", FUNC(gen)] call EFUNC(sys_rpc,addProcedure);
+["getTeamSpeakChannelName", FUNC(handleGetTeamSpeakChannelName)] call EFUNC(sys_rpc,addProcedure);
+["getServerName", FUNC(handleGetServerName)] call EFUNC(sys_rpc,addProcedure);
```

Comment:
Those should be `handleGet...` like the above ones. Follow naming of `getPluginVersion`.